guides/user/applications/create.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: single
+title:  "Create an application"
+sidebar:
+  nav: guides
+---
+
+You can't create a deployment pipeline unless you have an application to put it
+in. If you're accessing Spinnaker for the first time and there are no
+applications available, the first thing you're going to do is create one.
+
+1. From any screen, click on the **Applications** tab.
+
+   Note that there might be some (unconfigured) applications there already.
+   these might be created automatically, based on existing infrastructure, but
```

Comment:
I think this is a little confusing. (e.g., it's not clear what an unconfigured application is.) Suggest changing it to something like:

"Note that sometimes applications are automatically created based on your existing infrastructure, but these applications aren't suitable..." 

Might also be worth noting how to tell the difference between automatically created applications and pipeline-ready applications.

Reply:
I've reworded, but @lwander, ptal after I commit the changes, because I'm not sure I'm correctly characterizing the purpose of these inferred applications.